src/components/application_manager/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -143,6 +143,7 @@ void OnSystemRequestNotification::AddHeader(BinaryMessage& message) const {
 
   message.clear();
   message.assign(header.begin(), header.end());
+  message.push_back(0);
```

Comment:
@LevchenkoS As the `BinaryMessage` is a container for 'uint8_t' a.k.a. `unsigned char` it is better to be assigned a value `0u` if the intent is to add a termination character. If not a different value than zero should be considered.